app/Repositories/CacheStorage.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Gladiator\Repositories;
+
+use Illuminate\Support\Facades\Cache;
+
+trait CacheStorage
```

Comment:
Sorry, can you briefly explain why we need our own implementations of these methods? are there business logic reasons to define say a custom implementation of `Cache::flush()` or `Cache::get()`